react/tic-tac-toe/src/app/screens/Players/index.js

```diff
@@ -1,13 +1,93 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, Component } from 'react';
+import { connect } from 'react-redux';
+import { func, arrayOf } from 'prop-types';
 
+import getMatches from '../../../services/MatchesService';
+import actionsCreators from '../../../redux/match/actions';
 import TopBar from '../Game/components/TopBar';
+import { matchPropType } from '../../../constants/propTypes';
+import { PLAYER_ONE, TIE } from '../../../constants/gameConstants';
+import { history } from '../../../redux/store';
 
-function Players() {
-  return (
-    <Fragment>
-      <TopBar />
-    </Fragment>
-  );
+import styles from './styles.module.scss';
+
+
+class Players extends Component {
+  componentDidMount() {
+    if (this.props.matchHistory.length === 0) {
+      const { getMatchHistory } = this.props;
+      getMatches()
```

Comment:
Create an action to get the matches, with the corresponding success and failure actions too. 
Remember we usually always call the services via actions where we encapsulate all behavior.